core/chains/evm/config/toml/config.go

```diff
@@ -786,6 +787,11 @@ func (d *DAOracle) setFrom(f *DAOracle) {
 		d.OracleAddress = v
 	}
 	d.CustomGasPriceCalldata = f.CustomGasPriceCalldata
+	if f.L1ChainID == "" {
+		d.L1ChainID = "1"
```

Comment:
Not sure how I feel about a hard coded default here. I don't think we do this for any other config. Since you already have a value for this in `fallback.toml`, think this would never be empty string anyways